examples/impute/plot_missing_values.py

```diff
@@ -8,6 +8,9 @@
 The median is a more robust estimator for data with high magnitude variables
 which could dominate results (otherwise known as a 'long tail').
 
+With ``KNNImputer``, missing values can be imputed using the weighted
+or unweighted mean of the desired number of nearest neighbors.
+
```

Comment:
Explanation string here: instead of repeating 'round-robin' from the docstring of the imputer, perhaps replace by something more intuitively understandable, such as: column-by-column fashion